content/en/monitors/notify/variables.md

```diff
@@ -467,6 +503,15 @@ If your monitor starts failing on the `service:ad-server` group, the notificatio
 @slack-ad-server There is an ongoing issue with ad-server.
 ```
 
+If you build dynamic handles with attributes that might not be present, the variable renders empty in the notification message and the handle will be invalid. To avoid missing notifications, when using dynamic handles with these variables make sure to add a fallback handle:   
```

Comment:
```suggestion
When building dynamic handles with attributes that might not always be present, you may encounter issues with notification delivery. If an attribute is missing, the variable renders empty in the notification message, resulting in an invalid handle. 

To avoid missed notifications when using dynamic handles with these variables, make sure to add a fallback handle:
```